driver/makedepend.ml

```diff
@@ -13,9 +13,9 @@
 (*                                                                        *)
 (**************************************************************************)
 
+open Misc
```

Comment:
I suggest in `Misc` you add `Stdlib.String` and then (either via `open` or otherwise) use `String.Map` and `String.Set` instead of `StringMap` and `StringSet`; those latter two aliases in `Misc` could then be removed.  I would hope this change converges on a world where the stdlib modules gain support for sets and maps, and uniform naming can be used throughout.  (I have a patch for such stdlib support under development as I mentioned.)

Reply:
I just did that. That was extremely boring work. O_°